src/MissionParallelCatchup/parallel_catchup_helm/files/logarithmic_range_generator.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+floorSize=$LOGARITHMIC_FLOOR_LEDGERS
+overlapLedgers=$OVERLAP_LEDGERS
+startLedger=$(echo "$STARTING_LEDGER" | awk '{printf "%d", $1}')
+latestLedgerNum=$(echo "$LATEST_LEDGER_NUM" | awk '{printf "%d", $1}')
+numParallelism=$NUM_PARALLELISM
+
+echo "starting logarithmic range generationg with the following inputs: 
+floorSize=$floorSize
+overlapLedgers=$overlapLedgers
+startLedger=$startLedger
+latestLedgerNum=$latestLedgerNum
+numParallelism=$numParallelism"
+
+generate_uniform () {
```

Comment:
yes this is a routine very similar to `uniform_range_generator.sh`. 
I chose to make them separate so that they can be more cleanly separated and evolve at different pace, the `uniform_range_generator.sh` is identical logic to the current mission's [getCatchupRange](https://github.com/stellar/supercluster/blob/main/src/FSLibrary/MissionCatchupHelpers.fs#L185) and won't be changing much.

Reply:
Should we rename the function in the `logarithmic_range_generator.sh` to `generate_logarithmic`?
Right now we have a script called logarithmic that internally uses the `unifrom` function.

Reply:
The logarithmic range generator partitions the entire history into logarithmic sized chunks, within each chunk, it equally divides the ranges into number of workers, that's why it's calling generate_uniform here (in the outer scope it divides the remaining ledgers by half before calling generate_uniform). 
In other words the range sizes aren't fully logarithmic, but more like "piecewise logarithmic, locally uniform".